docs/scenarios/nfs-pv/README.md

```diff
@@ -9,5 +9,320 @@ This scenario walks through Migration of a stateful OpenShift app with Persisten
+
+```bash
+$ oc create -f mysql-persistent-template.yaml 
+namespace/mysql-persistent created
+secret/mysql created
+service/mysql created
+persistentvolumeclaim/mysql created
+deploymentconfig.apps.openshift.io/mysql created
+```
+
+Note that the template deploys our MySQL instance to a new namespace `mysql-persistent`.
+
+Verify that the MySQL deployment bound to one of your PVs successfully.
+
+```bash
+$ oc get PersistentVolumes
+NAME      CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM               
+pv1       100Gi      RWO,ROX,RWX    Retain           Available
+pv2       100Gi      RWO,ROX,RWX    Retain           Bound       mysql-persistent/mysql
+```
+
+---
+
+### 3. Populating MySQL with sample data
+
+Before performing a migration, let's populate our MySQL database with some data. Start by getting the pod name of the MySQL instance that we deployed in the previous step.
+
+```bash
+$ oc get pods -n mysql-persistent --selector "deployment=mysql-1"
+NAME            READY     STATUS    RESTARTS   AGE
+mysql-1-k985t   1/1       Running   0          35m
+```
+
+Now that we have the pod name, we can copy `data.sql` to `opt/app-root/src/data.sql` in our MySQL pod. We'll populate the database with some sample data this way.
+```bash
+$ oc cp -n mysql-persistent data.sql mysql-1-k985t:/opt/app-root/src
+```
+
+```
+$ oc exec -n mysql-persistent mysql-1-k985t -- /bin/bash -c "mysql -uMYSQL_USER -pMYSQL_PASSWORD MYSQL_DATABASE < /opt/app-root/src/data.sql"
+```
+
+Running data.sql against the database may take a while. After it exits, you can check the mounted NFS volume consumption.
+
+```
+$ oc exec -n mysql-persistent mysql-1-gpl2p -- /bin/bash -c df -h
+Filesystem                                1K-blocks  Used      Available  Use%  Mounted on
+123.234.321.210:/var/lib/nfs/exports/pv2  10473472   2881536   7591936    28%   /var/lib/mysql/data
+[...]
+```
+
+---
+
+### 4. Modifying MigPlan and MigMigration to migrate our NFS PV backed MySQL database
+
+To migrate our `mysql-persistent` namespace, we'll ensure that the `namespaces` field of our MigPlan includes mysql-persistent. 
+
+Modify the contents of [config/samples/mig-plan.yaml](https://github.com/fusor/mig-controller/blob/master/config/samples/mig-plan.yaml), adding 'mysql-persistent' to 'namespaces'.
+
+```yaml
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+spec:
+  # [!] Change namespaces to adjust which OpenShift namespaces should be migrated 
+  #     from source to destination cluster.
+  namespaces:
+  - mysql-persistent
+
+[... snipped, see config/samples/mig-plan.yaml for other required fields ...]
+```
+
+We also need to create a MigMigration with `quiescePods: true` since only one MySQL Pod can hold the lock at once, and we this scenario covers the 'move' PV action that re-mounts the *source cluster* NFS PVs on the *destination cluster*
```

Comment:
`and we this scenario` -> `and this scenario`

Reply:
+1 I noticed I did this but lost track of it.